package/MDAnalysis/coordinates/H5MD.py

```diff
@@ -1002,13 +1000,19 @@ class H5MDWriter(base.WriterBase):
             'kcal/(mol*Angstrom)': 'kcal mol-1 Angstrom-1',
             'kcal/(mol*A)': 'kcal mol-1 Angstrom-1'}}
 
+    @due.dcite(Doi("10.25080/majora-1b6fd038-005"),
+               description="MDAnalysis trajectory reader/writer of the H5MD"
+               "format", path=__name__)
+    @due.dcite(Doi("10.1016/j.cpc.2014.01.018"),
+               description="Specifications of the H5MD standard",
+               path=__name__, version='1.1')
     def __init__(self, filename, n_atoms, n_frames=None, driver=None,
                  comm=None, convert_units=True, chunks=None, compression=None,
                  compression_opts=None, positions=True, velocities=True,
                  forces=True, timeunit=None, lengthunit=None,
                  velocityunit=None, forceunit=None, author='N/A',
                  author_email=None, creator='MDAnalysis',
-                 creator_version=mda.__version__, **kwargs):
+                 creator_version=mda_version, **kwargs):
```

Comment:
`mda.__version__` looks ok to me, why did you change it?

Reply:
instead of importing all of `mda`, I did: `from MDAnalysis import __version__ as mda_version`, since the only thing I needed to import `mda` for was `mda.__version__`. Is one way preferable over the other?

Reply:
You will immediately import all of mda with `from MDAnalysis import __version__` — it makes no difference. Therefore, I'd keep it simple and clear (as it was previously). Generally, I am not a fan of `from x import y` because it is messy. "Namespaces are a honking good idea. Let's have more of those" is one of the Python Zens... IIRC.

Reply:
Still to do